### ISSUE TITLE
Commit repository with adequate structure

### ISSUE THREAD

**Issue by denis-kralj** [denis-kralj/reimagined-adventure#3]
In the current prototype repository a few things are missing to create an initial commit:

- .gitignore file
- LICENSE file
- README.md file
- build/watch commands for working on the project (backend files already work with this but not the client code)

**Comment by denis-kralj** [comment]
gitignore was simple enough, but running `dotnet watch run` in parallel with `webpack --mode=development --watch` seems to be non trivial, at least that is what search engines will lead you to believe :D

**Comment by denis-kralj** [comment]
Seems for now I will not pester myself with the one-liner `watch` command, it requires additional research. A new issue will be created.

**Comment by denis-kralj** [comment]
resolved and committed
